src/js/tech/middleware.js

```diff
@@ -166,7 +166,7 @@ export function mediate(middleware, tech, method, arg = null) {
   const callMethod = 'call' + toTitleCase(method);
   const middlewareValue = middleware.reduce(middlewareIterator(callMethod), arg);
   const terminated = middlewareValue === TERMINATOR;
-  const returnValue = terminated ? null : tech[method](middlewareValue);
+  const returnValue = terminated ? TERMINATOR : tech[method](middlewareValue);
```

Comment:
I also noticed that it is possible for our `player` values to be set to `null`, and now `TERMINATOR` if a `call<method>` middleware terminates. We may want to think about handling that.

Reply:
Would it be better to just return `TERMINATOR` for the `play` method? At least we can handle that?

Reply:
Can't we check for `null` specifically? Regular functions in javascript return `undefined` if don't have an explicit return value.

Reply:
let me see what ie 11 does, but it may be OK to check against null. I do think we may want to add a deprecation here for the future though.

Reply:
We return null again now.